README.md

```diff
@@ -0,0 +1,13 @@
+# Gonum diff  [![Build Status](https://travis-ci.org/gonum/diff.svg)](https://travis-ci.org/gonum/diff)  [![Coverage Status](https://img.shields.io/coveralls/gonum/diff.svg)](https://coveralls.io/r/gonum/diff)
+
+Quad provides numerical integration of single variable functions by quadrature for the Go language.
```

Comment:
Isn't quadrature a synonym for numerical integration? What about `Quad provides numerical evaluation of univariate definite integrals for the Go language` or ` ... of definite integrals of single-variable functions`?